src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -159,8 +163,8 @@ public function createRaAction(Request $request)
         $command->identityId  = $identityId;
         $command->institution = $raCandidate->institution;
 
-        $form = $this->createForm('ra_management_create_ra', $command)->handleRequest($request);
-        if ($form->isValid()) {
+        $form = $this->createForm(CreateRaType::class, $command)->handleRequest($request);
```

Comment:
This throws an exception because the FQDN couldn't be resolved

Reply:
Well spotted! Fixed this in 2f52466

Reply:
If I want to change the role of an Admin I now get an exception in the AnchorTye

Reply:
U2F isn't working for me while I'm registering. It's working on `https://demo.yubico.com/u2f?tab=register` in Firefox. I don't know if that is because of this upgrade or if it wasn't working before. If you would like I could test that?

Reply:
Hi @pablothedude, thanks for the AnchorType spot! Fixed this in e4bb21f 

Regarding the u2f question. I'm not aware if the feature worked in Firefox prior to my changes. To my knowledge it did not. Maybe @joostd or @pmeulen can verify this? And I'm not fully certain if this token type is in use at the moment.  FYI, the stepup u2f bundle is not at a stable release yet.

Reply:
Thanks for the changes of the AnchorType.